greeting-service/src/main/resources/additional-spring-configuration-metadata.jsoneeee

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
is this json file used? I can't see anywhere and the format `jsoneeee` seems wrong.

Reply:
removed the file